sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Supported creating `GalleryImageVersion` from source virtual machine by specifying `withSourceVirtualMachine`.
+
 ### Breaking Changes
```

Comment:
may need to delete the empty headings?

Reply:
Now changelog tool will do that when doing pre releasing PR.